validator/test/fixtures/gke_enable_workload_identity/assets/data.json

```diff
@@ -0,0 +1,162 @@
+            "createTime": "2016-11-19T05:58:02+00:00",
+            "currentMasterVersion": "1.10.11-gke.1",
+            "currentNodeCount": 4,
+            "currentNodeVersion": "1.4.6",
+            "endpoint": "104.196.229.72",
+            "initialClusterVersion": "1.4.6",
+            "instanceGroupUrls": [
+              "https://www.googleapis.com/compute/v1/projects/pso-cicd8/zones/us-west1-b/instanceGroupManagers/gke-canary-west-default-pool-496ebc1d-grp"
+            ],
+            "legacyAbac": {
+              "enabled": true
+            },
+            "location": "us-west1-b",
+            "locations": [
+              "us-west1-b"
+            ],
+            "loggingService": "logging.googleapis.com",
+            "monitoringService": "monitoring.googleapis.com",
+            "name": "canary-west",
+            "network": "default",
+            "networkConfig": {
+              "network": "projects/pso-cicd8/global/networks/default",
+              "subnetwork": "projects/pso-cicd8/regions/us-west1/subnetworks/default"
+            },
+            "nodeIpv4CidrSize": 24,
+            "nodePools": [
+              {
+                "autoscaling": {},
+                "config": {
+                  "diskSizeGb": 100,
+                  "imageType": "COS",
+                  "machineType": "n1-standard-1",
+                  "oauthScopes": [
+                    "https://www.googleapis.com/auth/compute",
+                    "https://www.googleapis.com/auth/devstorage.read_only",
+                    "https://www.googleapis.com/auth/logging.write",
+                    "https://www.googleapis.com/auth/servicecontrol",
+                    "https://www.googleapis.com/auth/service.management.readonly",
+                    "https://www.googleapis.com/auth/trace.append"
+                  ],
+                  "serviceAccount": "default"
+                },
+                "initialNodeCount": 2,
+                "instanceGroupUrls": [
+                  "https://www.googleapis.com/compute/v1/projects/pso-cicd8/zones/us-west1-b/instanceGroupManagers/gke-canary-west-default-pool-496ebc1d-grp"
+                ],
+                "management": {
+                  "autoUpgrade": "true"
+                },
+                "name": "default-pool",
+                "selfLink": "https://container.googleapis.com/v1/projects/pso-cicd8/zones/us-west1-b/clusters/canary-west/nodePools/default-pool",
+                "status": "RUNNING",
+                "version": "1.4.6"
+              }
+            ],
+            "selfLink": "https://container.googleapis.com/v1/projects/pso-cicd8/zones/us-west1-b/clusters/canary-west",
+            "servicesIpv4Cidr": "10.47.240.0/20",
+            "status": "RUNNING",
+            "subnetwork": "default",
+            "workloadIdentityConfig": {
+                "identityNamespace": "test-project-123-namespace"
+            },
+            "zone": "us-west1-b"
```

Comment:
We should also have a cluster to test with the following:

      "workloadIdentityConfig": {
        "workloadPool": "test-project-123.svc.id.goog"
      },